### DISCUSSION TITLE
Fail to import crc file

### DISCUSSION THREAD

**Discussion by Ericpan3** [mtex-toolbox/mtex#856] [Ask Anything] [answered]
I was trying to import the ".crc" file in the toolbox, but fail to open it and showing "Could not detect file format. Please contact one of the maintainers of MTEX and send him a copy of your files". 
So, do I need to send the file to someone?

**Comment by kilir** [comment]
Hi,
do you have the corresponding .cpr at the same location? These two go together.
Cheers,
Rüdiger

**Reply by Ericpan3** [reply]
Yes, I have that. Do I need to put them in the same folder when I import the .crc file?

**Comment by zmichels** [comment, marked as answer]
Yes, they need to be in the same location/folder from where you load.

**Reply by Ericpan3** [reply]
Thank you! It works now
